変数名の省略によるコンパイルエラー

このコードをコンパイルしようとした際、以下のエラーでコンパイルできませんでした。
```
sugi@Z690:~/jquery-practice$ gcc main.c 
In file included from main.c:2:
jquery.h: In function ‘appendTo’:
jquery.h:125:15: error: parameter name omitted
  125 | void appendTo(char*) {
      |               ^~~~~
```

該当の行の変数に適当な名前(`s`など)をつけてあげると正常にコンパイル・実行できました。`gcc std=c17`などいくつかの規格を試しましたが、元のコードではコンパイルできませんでした。環境は以下の通りです。
```
sugi@Z690:~/jquery-practice$ gcc -v
Using built-in specs.
COLLECT_GCC=gcc
COLLECT_LTO_WRAPPER=/usr/lib/gcc/x86_64-linux-gnu/9/lto-wrapper
OFFLOAD_TARGET_NAMES=nvptx-none:hsa
OFFLOAD_TARGET_DEFAULT=1
Target: x86_64-linux-gnu
Configured with: ../src/configure -v --with-pkgversion='Ubuntu 9.4.0-1ubuntu1~20.04.1' --with-bugurl=file:///usr/share/doc/gcc-9/README.Bugs --enable-languages=c,ada,c++,go,brig,d,fortran,objc,obj-c++,gm2 --prefix=/usr --with-gcc-major-version-only --program-suffix=-9 --program-prefix=x86_64-linux-gnu- --enable-shared --enable-linker-build-id --libexecdir=/usr/lib --without-included-gettext --enable-threads=posix --libdir=/usr/lib --enable-nls --enable-clocale=gnu --enable-libstdcxx-debug --enable-libstdcxx-time=yes --with-default-libstdcxx-abi=new --enable-gnu-unique-object --disable-vtable-verify --enable-plugin --enable-default-pie --with-system-zlib --with-target-system-zlib=auto --enable-objc-gc=auto --enable-multiarch --disable-werror --with-arch-32=i686 --with-abi=m64 --with-multilib-list=m32,m64,mx32 --enable-multilib --with-tune=generic --enable-offload-targets=nvptx-none=/build/gcc-9-Av3uEd/gcc-9-9.4.0/debian/tmp-nvptx/usr,hsa --without-cuda-driver --enable-checking=release --build=x86_64-linux-gnu --host=x86_64-linux-gnu --target=x86_64-linux-gnu
Thread model: posix
gcc version 9.4.0 (Ubuntu 9.4.0-1ubuntu1~20.04.1) 
```

--- a/jquery.h
+++ b/jquery.h
@@ -122,7 +122,7 @@ int col2seq(char *seq, char *col) {
 	return 0;
 }
 
-void appendTo(char*) {
+void appendTo(char* target) {
 	char seq[32] = {0};
 
 	if (state.style.bold)

